src/workloads/query/multiplanner/ClusteredCollection.yml

```diff
@@ -0,0 +1,410 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
```

Comment:
Update the test description to reflect how it tests with cluster index.

Reply:
Done